docs/resources/netflow_exporter_policy.md

```diff
@@ -0,0 +1,189 @@
+  source_ip_type         = "custom-src-ip"
+  source_ip_address      = "1.1.1.1/10"
+  version                = "v9"
+  relation_to_vrf = [
+    {
+      annotation = "annotation_1"
+      target_dn  = "uni/tn-test_tenant/ctx-test_vrf"
+    }
+  ]
+  relation_to_epg = [
+    {
+      annotation = "annotation_1"
+      target_dn  = "uni/tn-test_tenant/ap-test_ap/epg-test_epg"
+    }
+  ]
+  annotations = [
+    {
+      key   = "key_0"
+      value = "value_1"
+    }
+  ]
+  tags = [
+    {
+      key   = "key_0"
+      value = "value_1"
+    }
+  ]
+}
+
+```
+
+All examples for the Netflow Exporter Policy resource can be found in the [examples](https://github.com/CiscoDevNet/terraform-provider-aci/tree/master/examples/resources/aci_netflow_exporter_policy) folder.
+
+## Schema ##
+
+### Required ###
+
+* `destination_ip_address` (dstAddr) - (string) The destination IP address of the remote node.
+* `destination_port` (dstPort) - (string) The destination port of the remote node.
+  - Valid Values: `dns`, `ftpData`, `http`, `https`, `pop3`, `rtsp`, `smtp`, `ssh`, `unspecified`.
+* `name` (name) - (string) The name of the Netflow Exporter Policy object.
+
+### Read-Only ###
+
+* `id` - (string) The distinguished name (DN) of the Netflow Exporter Policy object.
+
+### Optional ###
+* `parent_dn` - (string) The distinguished name (DN) of the parent object, possible resources:
+  - [aci_tenant](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/tenant) ([fvTenant](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvTenant/overview))
+  - The distinguished name (DN) of classes below can be used but currently there is no available resource for it:
+    - [infraInfra](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/infraInfra/overview)
+
+  - Default: `uni/infra`
+  
+* `annotation` (annotation) - (string) The annotation of the Netflow Exporter Policy object.
+  - Default: `orchestrator:terraform`
+* `description` (descr) - (string) The description of the Netflow Exporter Policy object.
+* `qos_dscp_value` (dscp) - (string) The DSCP value of the Netflow Exporter Policy object.
+  - Default: `CS2`
+  - Valid Values:
+    * One of  `AF11`, `AF12`, `AF13`, `AF21`, `AF22`, `AF23`, `AF31`, `AF32`, `AF33`, `AF41`, `AF42`, `AF43`, `CS0`, `CS1`, `CS2`, `CS3`, `CS4`, `CS5`, `CS6`, `CS7`, `EF`, `VA`.
+    * Or a value in the range of `0` to `63`.
+* `name_alias` (nameAlias) - (string) The name alias of the Netflow Exporter Policy object.
+* `owner_key` (ownerKey) - (string) The key for enabling clients to own their data for entity correlation.
+* `owner_tag` (ownerTag) - (string) A tag for enabling clients to add their own data. For example, to indicate who created this object.
+* `source_ip_type` (sourceIpType) - (string) The type of the source IP address: It can be one of the available management IP address for a given leaf or a custom IP Address.
```

Comment:
Not sure I would add this description: `It can be one of the available management IP address for a given leaf or a custom IP Address`. Should we clean this up, and just keep the type part. The valid values already provide roughly the same information.

Reply:
changed it.